realm/src/androidTest/java/io/realm/RealmListTest.java

```diff
@@ -47,6 +49,143 @@ protected void tearDown() throws Exception {
         testRealm.close();
     }
 
+    public void testPublicNoArgConstructor() {
```

Comment:
A big fat comment section //******************* etc
describing that we test the non managed mode would ease the overview.